frontend/src/app/main/data/workspace/libraries.cljs

```diff
@@ -1389,41 +1426,36 @@
        :file-id file-id
        :library-id library-id})
 
-    ;; NOTE: this event implements UpdateEvent protocol for perform an
-    ;; optimistic update state for make the UI feel more responsive.
-    ptk/UpdateEvent
-    (update [_ state]
-      (let [libraries (:workspace-shared-files state)
```

Comment:
:workspace-shared-files doesn't exist since ages ago

This wasn't doing anything, and the UI is quite responsive without it